content/sidebars.yaml

```
@@ -660,7 +660,7 @@
       - group: Dev Tools
         expanded: false
         items:
-          - label: RPC Tool
+          - page: /dev-tools/rpc-tool.page.tsx
```

The label should be the same as before :)
```suggestion
          - label: RPC Tool
             page: /dev-tools/rpc-tool.page.tsx
```